content/reference/metrics.md

```diff
@@ -9,7 +9,9 @@ This document describes iter8's out-of-the-box metrics and how to extend iter8's
 
 ## Iter8's out-of-the-box metrics
 
-Iter8 leverages the metrics collected by Istio telemetry and stored in Prometheus. Users relying on iter8's out-of-the-box metrics can simply reference them in the criteria section of an _experiment_ specification, as illustrated in [this tutorial](thistutorial.md) and documented in the [`Experiment` CRD documentation](../experiment). Iter8's out-of-the-box metrics are as follows.
+<!-- TODO: What is thistutorial.md? -->
+
+Iter8 leverages the metrics collected by Istio telemetry and stored in Prometheus. Users relying on iter8's out-of-the-box metrics can simply reference them in the criteria section of an _experiment_ specification, as illustrated in [this tutorial](thistutorial.md) and documented in the [`Experiment` CRD documentation]({{< ref "experiment" >}}). Iter8's out-of-the-box metrics are as follows.
```

Comment:
```
[this tutorial](thistutorial.md)
```

Can someone fix this? This is an example of how `ref`/`relref` can be useful as it'll inform the user that this page does not exist.